### ISSUE TITLE
Specific Avatar Import Crashes Resonite

### ISSUE THREAD

**Issue by Lv100Latios** [Yellow-Dog-Man/Resonite-Issues#593]
### Describe the bug?

Upon loading the Rexouium Evolved avatar (found here: https://xtosca.gumroad.com/l/ToscaRex?layout=profile) in Resonite, selecting the defaults for avatar models, and auto humanoid, upon trying to decode the .fbx file, the game suddenly freezes and then crashes without any apparent error handling.

Avatar cannot be provided due to being a paid asset. Importing a different avatar (Nardoragon) does not crash the game. Unsure what's causing it.

Note - all mod loader files were removed before attempting to recplicate the crash - forgot to remove the parameter in steam, though.

### To Reproduce

On the current build of the game, try to import the Rexouium Evolved .fbx with all of the defaults selected and using auto humanoid for scale. When the avatar gets to decoding, it should crash.

### Expected behavior

The avatar should import as expected.

### Resonite Version Number

Beta 2023.11.8.335

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Index

### Log Files

[NO-1 - 2023.11.8.335 - 2023-11-08 20_27_46.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13303855/NO-1.-.2023.11.8.335.-.2023-11-08.20_27_46.log)
[Player.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13303856/Player.log)

### Additional Context

Importing the avatar in a previous build did not crash the client (approx 1 week ago, avatar bought on 2 November 2023).

### Reporters

U-Alice / alicederg (Discord)

**Comment by Stellanora64** [comment]
Does it crash if it's exported as other formats? (.gltf, .glb, or .obj)

I also have seen this avatar in Resonite before, so the most likely cause is the recent update to assimp.

**Comment by Lv100Latios** [comment]
> Does it crash if it's exported as other formats? (.gltf, .glb, or .obj)
> 
> I also have seen this avatar in Resonite before, so the most likely cause is the recent update to assimp.

Nope! Exporting the avatar as a GLB in Blender will allow the asset to be imported as normal. Unsure why the FBX of this avatar is causing an issue when it's not elsewhere. And since the logs don't say anything... idk.
